rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -33,19 +42,34 @@ final class AnnotateThrowablesRector extends AbstractRector
     /**
      * @var string
      */
-    private const RETURN_DOCBLOCK_TAG_REGEX = '#@return[ a-zA-Z0-9\|\\\t]+#';
+    private const RETURN_DOCBLOCK_TAG_REGEX = '#@return[ a-zA-Z0-9-_\|\\\t]+#';
+
+    /**
+     * @var string
```

Comment:
Could you add http://regex101.com/ link? Just to make future refactoring easier

Reply:
Do you mean a link?

Reply:
Done